.github/workflows/python.yaml

```diff
@@ -40,13 +40,30 @@ jobs:
           poetry install --extras "all"
       - name: Run tests
         if: always()
-        # Leave the stdout output active but also extract the coverage report into a txt file
-        run: poetry run pytest --cov=blackboxopt tests | tee /dev/tty | sed -n '/ -----------/,$p' > coverage.txt
+        run: poetry run pytest --cov=blackboxopt tests | sed -n '/ -----------/,$p' > coverage.txt
```

Comment:
I needed to remove the pipe to stdout because `/dev/tty` doesn't seem to be available on all github actions runners.